packages/components/src/popover/index.js

```diff
@@ -61,7 +61,13 @@ class Popover extends Component {
 	componentDidMount() {
 		this.toggleWindowEvents( true );
 		this.refresh();
-		this.focus();
+		/*
+		 * Without the setTimeout, the dom node is not being focused. Related:
```

Comment:
I know you didn't author it, but the original comment reads to me as "I encountered an issue, couldn't figure out why it was happening, but by pure chance this alternative seems to do what I want"; we should strive for a higher level of quality.